chart/identity-manager-sts/templates/configMap.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.name" . }}-app-config
+data:
+  {{- with .Values.identityManagerSTS.config }}
+  sts.yaml: |
+    server:
+      listen: {{ .server.listen }}
+    otel:
+      enabled: {{ .otel.enabled }}
```

Comment:
I would default this value in particular so you can ensure you don't hit a parse error or nil pointer at run time

Reply:
Done, also put the rest of the block within an if to remove it if disabled.